laze-project.yml

```diff
@@ -812,6 +812,29 @@ modules:
         FEATURES:
           - ariel-os/coap-server
 
+  - name: coap-server-config-unprotected
+    help: Configure the CoAP server to accept any request without authorization checks.
+
+      The device will have no cryptographic identity for CoAP, and not even
+      accept opportunistic EDHOC.
+    depends:
+      - coap
+    env:
+      global:
+        FEATURES:
+          - ariel-os/coap-server-config-unprotected
+
+  - name: coap-server-config-demokeys
+    help: Configure the CoAP server as expected by simple examples that expect a fixed server key.
+
+      This configuration may change over time, in lockstep with the examples' setup.
+    depends:
+      - coap
+    env:
+      global:
+        FEATURES:
+          - ariel-os/coap-server-config-demokeys
+
```

Comment:
Is there better practice than just having multiple modules? Is there a way to express that a module conflicts with any other of a group?